.github/workflows/pr_title_checker.yml

```diff
@@ -0,0 +1,29 @@
+name: Ensure PR title has a valid prefix
+
+on:
+  pull_request:
+    branches:
+      - main
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@c6295a65d1254861815972266d5933fd6e532bdf # v2.11.1
+        with:
+          disable-sudo: true
+          egress-policy: audit
+      - name: Check PR title    
+        uses: thehanimo/pr-title-checker@v1.4.3
```

Comment:
Need to use the commit hash here or our security scan will get mad.

Reply:
Already fixed